### DISCUSSION TITLE
activationMode = "none" in Editable

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [chakra-ui/chakra-ui#9260] [Ideas]
In Chakra there is not the "none" value for activationMode in Editable
I wanted to be able to only enter the edit mode when clicking the edit button. So clicking twice in the text would select the entire text and not enter the editing state.
In Zag is possible: https://zagjs.com/components/editable

**Comment by segunadebayo** [comment]
Pass `activationMode` through. Ark already supports it on Editable.
